src/Nethereum.RPC/Eth/Blocks/LastConfirmedBlockNumberService.cs

```diff
@@ -19,7 +19,7 @@ public class LastConfirmedBlockNumberService : ILastConfirmedBlockNumberService
 
         public LastConfirmedBlockNumberService(
             IEthBlockNumber ethBlockNumber,
-            uint minimumBlockConfirmations = DEFAULT_BLOCK_CONFIRMATIONS,
```

Comment:
This was fine

Reply:
There is no need for an extra overload.

Reply:
I found I was repeating the default logic in the caller overloads - by allowing the nullable the default is only in one place, in the service that owns it.

Reply:
Can you elaborate on why the nullable is an issue?

Reply:
Oh for simplicity sake, and to let users know that if the don't use that the value will be the one in Default block confirmations, not 0

Reply:
Now if you are having issues with the overloads, you can create a specific overload not using default values, but using this(Default number) instead.

Reply:
Lots of tidying up this morning.  Default confirmations on interface so they are clear to the user and don't have to be duplicated elsewhere.  Added new overload for the orchestrator to take a single processor steps as well as an array of them.  (typical usage is just one set of steps but we now allow for both)

Reply:
Cool Ill merge now.. and we can change later.. we will do a review before release anyway.